python-sdk/src/astro/sql/__init__.py

```diff
@@ -4,13 +4,13 @@
 
 from astro.sql.operators.append import AppendOperator, append
 from astro.sql.operators.cleanup import CleanupOperator, cleanup
-from astro.sql.operators.data_validations.ColumnCheckOperator import (  # skipcq: PY-W2000
+from astro.sql.operators.data_validations.check_column import (  # skipcq: PY-W2000
     ColumnCheckOperator,
     column_check,
 )
-from astro.sql.operators.data_validations.SQLCheckOperator import (  # skipcq: PY-W2000
+from astro.sql.operators.data_validations.check_table import (  # skipcq: PY-W2000
     SQLCheckOperator,
-    sql_check,
+    check_table,
```

Comment:
Why are we calling this `check_table` and in `check_column` we call it `column_check`? Let's be consistent here.